libs/block-template-html/block-schema.json

```diff
@@ -1,11 +1,14 @@
 {
+  "$id": "http://example.com/types/entity-type/example/v/1",
```

Comment:
Ditto other comment – could use `Thing` from the other templates

Reply:
In fact we should probably just delete this file, it is a relic from when blocks bundled their schemas